cmd/geth/js.go

```diff
@@ -425,12 +437,13 @@ func (self *jsre) interactive() {
 	}
 }
 
-func hidepassword(input string) string {
-	if passwordRegexp.MatchString(input) {
-		return passwordRepl
-	} else {
-		return input
+func postprocess(input string) (string, bool) {
```

Comment:
This function doesn't do any real post processing and the returned string is not actually necessary. Consider changing it to something like `func mustLogInHistory(input string) bool`.